Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :test do
```

Comment:
Totally, I am unsure how it gets loaded though. 

Would it always be loaded even though we are never using pry? Would it slow down the spec suite?